### PR TITLE
Propose OSC as an OpenSearch partner

### PR DESCRIPTION
**Pull request by flaxsearch**
### Description
[Describe what this change achieves]
 
### Issues Resolved
[List any issues this PR will resolve]

### Check List
- [ ] Commits are signed per the DCO using --signoff

By submitting this pull request, I confirm that my contribution is made under the terms of the BSD-3-Clause License.

### CODE REVIEW

**Comment by flaxsearch** [comment]
I'll need to get someone to upload our logo, sorry!

**Comment by stockholmux** [comment]
@flaxsearch This is awesome! 

Make sure you add your logo! Also, looks like you missed the DCO sign off too - so make sure to fix that when you add it!

**Comment by elfisher** [comment]
@flaxsearch super excited to see OSC! Let us know if you have having any issues with the DCO sign off and adding a logo.

Thanks!

File: _partners/opensourceconnections.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+---
+name: OpenSource Connections
+# upload your logo to the following directory - must be square
+logo: '/assets/media/partners/osc-logomark-color.png'
```
Comment (krisfreedain) [inline _partners/opensourceconnections.md:4]: hello @flaxsearch - please upload the osc-logomark-color.png to '/assets/media/partners/' as part of this pull request. thanks

**Review by krisfreedain — CHANGES_REQUESTED** [review]
@flaxsearch - small change needed. missing image.

File: _partners/opensourceconnections.md (hunk 1 above)
Comment (krisfreedain) [reply _partners/opensourceconnections.md:4]: @flaxsearch - wanted to follow up on this one - please let us know

**Comment by flaxsearch** [comment]
OSC-logomark-color(1)

File: _partners/opensourceconnections.md (hunk 1 above)
Comment (flaxsearch) [reply _partners/opensourceconnections.md:4]: hey Kris having some trouble adding it to that folder, could you commit the attached for me?

**Comment by flaxsearch** [comment]
HI @krisfreedain can you help?

**Comment by krisfreedain** [comment]
> HI @krisfreedain can you help?
@flaxsearch 
I'll create a new PR with your changes and take care of it - will tag you in that one ASAP for your approval

**Comment by krisfreedain** [comment]
closing - this is resolved via https://github.com/opensearch-project/project-website/pull/878

**Pull request closed without merging**